pkg/operator/controller/callbacks.go

```diff
@@ -170,7 +171,6 @@ func reconcileCreatePrometheusInfra(args *callbacks.ReconcileCallbackArgs) error
 		args.Recorder.Event(cr, corev1.EventTypeWarning, createResourceFailed, fmt.Sprintf("Failed to ensure prometheus resources exists, %v", err))
 		return err
 	}
-
```

Comment:
intended? afaik we mostly keep the empty line before returns

Reply:
ok :)